maps/graph_edges/base_graph/edges_hos.py

```diff
@@ -89,11 +91,11 @@
     {"from": {"map": "HOS_06", "id": "ShopBadgeA"},  "to": {"map": "HOS_06", "id": "ShopBadgeN"},  "reqs": []}, #+ SHARED REQUIREMENTS -> ShopBadgeN (FlowerSaverA)
     {"from": {"map": "HOS_06", "id": "ShopBadgeA"},  "to": {"map": "HOS_06", "id": "ShopBadgeO"},  "reqs": []}, #+ SHARED REQUIREMENTS -> ShopBadgeO (PowerPlusA)
     {"from": {"map": "HOS_06", "id": 0},             "to": {"map": "HOS_06", "id": "ShopRewardA"}, "reqs": [[{"starpieces": 10}],["can_climb_steps"]]}, #* Merluvlee's House Exit Left -> ShopRewardA (Mushroom)
-    {"from": {"map": "HOS_06", "id": "ShopRewardA"}, "to": {"map": "HOS_06", "id": "ShopRewardB"}, "reqs": [[{"starpieces": 23}]]}, #+ CHAINED REQUIREMENTS (SuperShroom)
+    {"from": {"map": "HOS_06", "id": "ShopRewardA"}, "to": {"map": "HOS_06", "id": "ShopRewardB"}, "reqs": [[{"starpieces": 22}]]}, #+ CHAINED REQUIREMENTS (SuperShroom)
```

Comment:
I may have missed the memo, why do the starpiece requirements change?

Reply:
The original calculation for the minimum of required star pieces to guarantee no softlocks used wrong numbers.
I noticed this because we originally didn't calculate the numbers ourselves, instead Mr.  Cheeze provided them and I didn't check them.
I just felt like checking the numbers was the right thing to do, and then noticed that the numbers were wrong